airbyte-integrations/connectors/source-zendesk-talk/source_zendesk_talk/schemas/ivr_routes.json

```diff
@@ -27,7 +27,7 @@
       "type": ["null", "object"]
     },
     "overflow_options": {
-      "type": ["null", "object"]
+      "type": ["null", "array"]
```

Comment:
do we know the schema/type of array elements?

Reply:
same as above